app/service/authorisation.py

```diff
@@ -22,15 +22,15 @@ def http_method_to_operation(method: str) -> AuthOperation:
     raise AuthorisationError(f"Unknown HTTP method {method}")
 
 
-def path_to_entity(path: str) -> AuthEntity:
+def path_to_entity(path: str) -> AuthEndpoint:
```

Comment:
Tiny niggle - but fix it in the next PR .... we can change the function name to something like `path_to_auth_endpoint()`

Reply:
Pushed to this PR 👍